compiler/one-cmds/onnx_legalizer.py

```diff
@@ -195,20 +320,20 @@ def transform_unidirectional_RNN(transformer, original_node, x, tensor_infos, ac
     outputs = original_node.output
     if direction == 'reverse':
         x.reverse()
-    w = transformer.make_node('Squeeze', [inputs[1]], 1, axes=[0])
-    r = transformer.make_node('Squeeze', [inputs[2]], 1, axes=[0])
+    w = transformer.make_squeeze(inputs[1], axes=[0])
```

Comment:
`make_sqeuuze` returns `self.make_node('Squeeze', [input], 1, axes=axes)[0]` and original code was make_node('Squeeze', [inputs[1]], 1, axes=[0])`, which doesn't have `[0]` at the end of it. Is it okay?

Reply:
`[0]` : return is list :)

Reply:
@seanshpark Umm.. I don't get it. You mean, type of `self.make_node('Squeeze', [input], 1, axes=axes)[0]` is a list?

Original type of `w`(`w = transformer.make_node('Squeeze', [inputs[1]], 1, axes=[0])`) is a list of string or int. And,  Changed one is just a string as the method spec said.

Reply:
Hum... I thought `self.make_node('Squeeze', [input], 1, axes=axes)` returned list and `[0]` is to access the item... not sure my understanding is correct -_-;

Reply:
@binarman , ping ?

Reply:
@mhs4670go
Sorry for the long delay.

**First thing: detailed description of what `make_node` method is doing.**

`make_node` is a generic method that should be capable to create nodes with multiple outputs.
In order to do this, it return array of strings(names of output tensors), instead of simple string.

In fact, this method works in two modes, depending on what is the type of `output` method:
- **int**: `output` contains number of outputs operation produces and method will create unique names automatically
- **list of strings**: `output` contains names of already existing tensors, that should be used as-is.

Second mode is used to attach created operations to original model.


**What changed**

Old code is mostly uses arrays of tensor names, new code mostly uses single tensor name for every argument.

Regarding this particular place:

- In old code `make_node('Squeeze')` created array, which is passed to `generate_one_direction_RNN` and then to other `make_node` methods.
- In this PR all values in this chain of calls are single string instead of array.

P.s.
Thanks to you comment I realized that documentation for `generate_one_direction_RNN` and `generate_one_direction_LSTM` is outdated =)